iam_check/test/multiple_policies/test.tf

```diff
@@ -7,6 +7,10 @@ terraform {
   }
 }
 
+provider "aws" {
```

Comment:
tf 1.5.7 was complaining about not having a provider configured when i ran this with the new conditions below